lms/static/js/spec/search/search_spec.js

```diff
@@ -89,6 +89,7 @@ define([
 
         beforeEach(function () {
             TemplateHelpers.installTemplate('templates/courseware_search/search_item');
+            TemplateHelpers.installTemplate('templates/courseware_search/search_item_seq');
```

Comment:
Hm, seems to me like it's better to use separate files and keep logic out of templates. Adding more result types would make things even messier.